canonical/react-components
# fix: Fix broken type declarations downstream imports

## Done

- Reversion of #1129 to fix broken absolute type imports.

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Checkout this PR
- Link packages with `yarn link-packages` in the react components repo.
- Checkout some project that depends on React Components.
- Try the following imports and verify they work as expected.

```tsx
import { MainTable } from "@canonical/react-components"; // a component
import { MainTableRow } from "@canonical/react-components/dist/components/MainTable/MainTable"; // a type
```

### Percy steps

- No visual changes expected.

## Fixes

Fixes: https://chat.canonical.com/canonical/pl/unjxx7ab8pgf3n9dncb7ir8anc
